src/LinearMaps.jl

```diff
@@ -10,6 +10,8 @@ using SparseArrays
 
 import Statistics: mean
 
+import ChainRulesCore: rrule, frule, unthunk, @thunk, NoTangent
```

Comment:
I think you need to `import` only those functions that you wish to overwrite, in this case `rrule`. As for the rest, I think `using ChainRulesCore` should be fine, or, if you want to be more restrictive, then `using ChainRulesCore: unthunk, @thunk, NoTangent`. `frule` doesn't seem to be used anywhere.